### ISSUE TITLE
RancherOS: No recovery switching to persistent console with no space left of device

### ISSUE THREAD

**Issue by 2devnull** [rancher/os#667]
Tried to switch to ubuntu console. Rebooted and got the following error:

![](http://oi64.tinypic.com/14l7jb5.jpg)
![](http://oi67.tinypic.com/wv4pj4.jpg)

It would have been nice to know that there was not enough disk space and therefore have RancherOS not accept this change. As it stands, how do I get it to successfully boot? i.e.: how to tell it to not try and install the ubuntu console so I can increase the disk size first?

**Comment by ibuildthecloud** [comment]
@2devnull The ubuntu console is not very large, couple hundred mb.  Are you running from memory or from disk?

**Comment by 2devnull** [comment]
running from disk. Will try getting in with your suggestion from #662.

**Comment by 2devnull** [comment]
ok, was able to change the ubuntu-console to false after booting with the following parameters:
rancher.state.dev=none
rancher.password=rancher

then mounting the /dev/sda1 to /home/rancher/disk

and changing true to false in this file:
```
[rancher@rancher disk]$ sudo cat /home/rancher/disk/var/lib/rancher/conf/cloud-config.yml 
rancher:
  services_include:
    ubuntu-console: true
```

**Comment by pwFoo** [comment]
I activated debian console on a apu mini server (console only). After reboot I experienced some some conection problems (latency, connectivity, unreachable, ...). Some reboots later I logged in, disabled debian console and all works fine again?
Is there a bug with the persistent consoles? 

I tried to install the latest docker 1.10, but no chance to install or just download it because of my network problems. I don't know if it's apu / hw or RancherOS / persistent console related.

Maybe related to this topic but I can't take a look at the error message at boot.
